news/2024/2024-09-01-featured-artist-track-updates-cygnus.md

```diff
@@ -0,0 +1,101 @@
+
+Anyway, let's get into the music already:
+
+## osu! original
+
+First off, we need to highlight the newest addition to **cygnus**'s discography: [Tree of Misery](https://osu.ppy.sh/beatmapsets/2242640), produced specifically for this year's [osu!mania 4K World Cup 2024](/wiki/Tournaments/MWC/2024_4K)!
+
+If you missed it on the mappool showcase today, you're in luck — it's also right here:
+
+<div align="center">
+    <video width="95%" controls>
+        <source src="https://assets.ppy.sh/media/news/cygnus_original_tree_of_misery.mp4" type="video/mp4" preload="none">
+    </video>
+</div>
+
+## Other tracks
+
+There's a lot more music than just that, though. **cygnus** released his newest album *Sorrow Mania* today too (every track of which is now here on osu!).
+
+All of those songs, plus a bunch more from earlier releases can be found on [**cygnus**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/359) with the **NEW** tag. Download a timed `.osz` file for whichever song will be your next map.
+
+If you're not in the mapping mood quite yet, play through any of these to get some inspiration:
+
+### cygnus - Bernkastel's Sanctuary
+
+[Here's another map](https://osu.ppy.sh/beatmapsets/2134400) hosted by [the artist himself](https://osu.ppy.sh/users/5052899)!
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/359/Midnight%20Sacrifice/cygnus%20-%20Bernkastel%27s%20Sanctuary.mp3">
+</audio>
+
+### cygnus - M U T A T i O N (((((9^9)^9)^9)^9)^9)
+
+Play through [this recently Ranked map](https://osu.ppy.sh/beatmapsets/2235017) hosted by [dopaminos](https://osu.ppy.sh/users/6570161)!
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/359/Sorrow%20Mania/cygnus%20-%20M%20U%20T%20A%20T%20i%20O%20N%20(((((9%5E9)%5E9)%5E9)%5E9)%5E9).mp3">
+</audio>
+
+### cygnus - Bloody Mary
+
+Today is a good day for osu!. [fanzhen0019](https://osu.ppy.sh/users/418699) uploaded [a new map](https://osu.ppy.sh/beatmapsets/2242344).
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/359/Sorrow%20Mania/cygnus%20-%20Bloody%20Mary.mp3">
+</audio>
+
+### cygnus - loppi_complex
+
+Experience a bit of **cygnus**'s new album through [this brand new beatmap](https://osu.ppy.sh/beatmapsets/2242440) hosted by the one and only [mithew](https://osu.ppy.sh/users/2571893)!
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/359/Sorrow%20Mania/cygnus%20-%20loppi%27complex.mp3">
+</audio>
+
+### cygnus - magic rock "Moola"
+
+Take your pick of [any of these 10 difficulties](https://osu.ppy.sh/beatmapsets/2105573) hosted by [Matrix](https://osu.ppy.sh/users/5052899)!
```

Comment:
the first time they are mentioned as "the artist himself", but here their osu! name is mentioned which could mislead people to thinking that they are different people?

Reply:
adjusted